modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -79,19 +79,55 @@ private function generateCallbackSecret() {
 	 */
 	public function getApiKey() {
 		return Capsule::table('tblpaymentgateways')
-			->where('gateway', 'blockonomics')
-			->where('setting', 'ApiKey')
-			->value('value');
+				->where('gateway', 'blockonomics')
+				->where('setting', 'ApiKey')
+				->value('value');
 	}
 
 	/*
-	 * Get user configured API key from database
+	 * Get list of crypto currencies supported by Blockonomics
 	 */
-	public function getAltcoins() {
-		return Capsule::table('tblpaymentgateways')
-			->where('gateway', 'blockonomics')
-			->where('setting', 'Altcoins')
-			->value('value');
+	public function getSupportedCurrencies() {
+		$all_currencies = array(
+  			'currencies' => array(
+  				array(
+			        'code' => 'btc',
+			        'name' => 'Bitcoin',
+			        'uri' => 'bitcoin'
+  				),
+    			array(
+			        'code' => 'bch',
+			        'name' => 'Bitcoin Cash',
+			        'uri' => 'bitcoincash'
+  				)
+  			)
+  		);
+		return json_encode($all_currencies);
```

Comment:
Why have overhead of *json_encode* when all calling functions are doing *json_decode* ?

Reply:
Removed in 5138aa5